src/app/resourcebundles/data/consumption/en.properties

```diff
@@ -488,6 +488,11 @@ frmelmnts.lbl.userId = User ID
 frmelmnts.lbl.userType = User Type
 frmelmnts.lbl.username = User Name
 frmelmnts.lbl.validPassword = Enter a valid password
+frmelmnts.lbl.charError = Password must contain at least 8 alphanumeric characters.
```

Comment:
use the below key & follow the same for all keys
frmelmnts.lbl.**password**.lwcsError